whois-commons/src/test/java/net/ripe/db/whois/common/dao/jdbc/DatabaseHelper.java

```diff
@@ -541,7 +550,7 @@ private static void validateFilePerTable(final JdbcTemplate jdbcTemplate) {
             return rs.getBoolean(1);
         });
 
-        if (filePerTable) {
+        if (Boolean.TRUE.equals(filePerTable)) {
```

Comment:
Don't allow `filePerTable` to be null (warn or fail)

Reply:
added extra check